### ISSUE TITLE
Healpix model registry tests add about 2m to CPU test time

### ISSUE THREAD

**Issue by oliverwm1** [ai2cm/ace#406]
Based on the tests running via CI, #1016 added about 2 minutes to the CPU runtime of our test suite, nearly doubling the total duration. Can we speed these tests up / reduce parameterization without losing coverage?

Compare the duration of "Run pytest" step in [commit prior](https://github.com/ai2cm/full-model/actions/runs/10046446413/job/27765924037) and [commit after it was merged](https://github.com/ai2cm/full-model/actions/runs/10066356241/job/27827557143). Can see a detailed breakdown of the slowest tests by opening that section and scrolling to end of logs:

<img width="852" alt="Screenshot 2024-07-25 at 9 14 22 AM" src="https://github.com/user-attachments/assets/ca047be4-57d1-485d-8eba-8b9353629762">

**Comment by anna-dodson** [comment]
Agreed this is not great. Could we either parallelize tests, or test them on gpu only (if that's faster)?

**Comment by oliverwm1** [comment]
Do you have a sense of why this test is so slow? In my experience it shouldn't take that long to make some toy data and run it through a pytorch module. The tests fail on my mac, so hard for me to troubleshoot. Can just make the data/model size for the test smaller?

**Comment by mcgibbon** [comment]
This is fixed in main, presumably by ai2cm/full-model#1065.

```
=================================================== slowest 20 durations ===================================================
35.16s call     fme/downscaling/test_train.py::test_resume_two_workers
32.23s call     fme/ace/test_train.py::test_resume_two_workers[SphericalFourierNeuralOperatorNet]
23.65s call     fme/ace/test_train.py::test_train_and_inference_inline[HEALPixRecUNet]
19.35s call     fme/downscaling/test_train.py::test_resume
11.51s call     fme/downscaling/test_train.py::test_train_main
9.80s call     fme/ace/test_train.py::test_train_and_inference_inline[SphericalFourierNeuralOperatorNet]
9.13s call     fme/ace/test_train.py::test_train_and_inference_inline[SFNO-v0.1.0]
7.49s call     fme/ace/test_train.py::test_copy_weights_after_batch[SphericalFourierNeuralOperatorNet]
7.18s call     fme/ace/test_train.py::test_resume[SphericalFourierNeuralOperatorNet]
5.33s call     fme/downscaling/test_train.py::test_train_eval_modes
4.90s call     fme/ace/registry/test_hpx.py::test_HEALPixRecUNet_reset
3.78s call     fme/ace/test_train.py::test_fine_tuning[SphericalFourierNeuralOperatorNet]
2.80s call     fme/core/aggregator/one_step/test_main.py::test_derived
2.40s call     fme/ace/inference/test_evaluator.py::test_inference_plus_one_model[False-54]
2.09s call     fme/ace/registry/test_hpx.py::test_HEALPixRecUNet_forward[inputs_config0-3-1-3-2-4-2-16]
2.06s call     fme/ace/registry/test_hpx.py::test_HEALPixRecUNet_forward[inputs_config1-3-0-3-2-4-2-16]
2.06s call     fme/ace/registry/test_hpx.py::test_HEALPixRecUNet_forward[inputs_config2-3-1-3-2-4-0-16]
1.33s call     fme/downscaling/test_train.py::test_restore_checkpoint
1.33s call     fme/ace/inference/test_evaluator.py::test_inference_data_time_coarsening
1.24s call     fme/downscaling/test_evaluator.py::test_evaluator_runs[checkpoint]
```
